openapi/src/main/scala/endpoints/openapi/DocumentedEndpoints.scala

```diff
@@ -0,0 +1,64 @@
+package endpoints
+package openapi
+
+import endpoints.algebra.MuxRequest
+
+/**
+  * Interpreter for [[algebra.DocumentedEndpoints]] that produces
+  * an [[OpenApi]] instance for endpoints.
+  */
+trait DocumentedEndpoints
+  extends algebra.DocumentedEndpoints
+    with DocumentedRequests
+    with DocumentedResponses {
```

Comment:
Finally, this is the interpreter that builds an OpenAPI description of the endpoint descriptions.